spec/features/download_layer_spec.rb

```diff
@@ -11,14 +11,20 @@
     allow(Geoblacklight::KmzDownload).to receive(:new).and_return(kmz_download)
   end
 
+  # cleanup any downloaded files
+  after do
+    FileUtils.rm("tufts-cambridgegrid100-04-shapefile.zip", force: true)
+    FileUtils.rm("tufts-cambridgegrid100-04-kmz.kmz", force: true)
+  end
+
   scenario "clicking initial shapefile download button should trigger download", js: true do
-    expect(shapefile_download).to receive(:get).and_return("mit-f6rqs4ucovjk2-shapefile.zip")
-    visit solr_document_path("mit-f6rqs4ucovjk2")
+    expect(shapefile_download).to receive(:get).and_return("tufts-cambridgegrid100-04-shapefile.zip")
+    visit solr_document_path("tufts-cambridgegrid100-04")
```

Comment:
this tufts-cambridge fixture actually has files available to download in the fixtures, and since we auto-click to download, we should use fixtures where this works